docs/architecture/adr-019-subspace-post-moving.md

```diff
@@ -0,0 +1,77 @@
+# ADR 019: Subspace post migration
+
+## Changelog
+- April 20th, 2023: First draft;
+
+## Status
+
+Proposed
+
+## Abstract
+
+This ADR introduces a new feature that enables post owners to move their own content between subspaces.
+
+## Context
+
+Currently, the only way to move a post to another subspace is by following process:
+1. Create an identical post as the original post in the target subspace
+2. Delete the original post on source subspace
+
+This process is overly complicated and time-consuming because it requires the user to perform two steps and manually recreate the post in the new location. Additionally, it can lead to inconsistencies if the user forgets to include certain details. A more user-friendly process would be beneficial for improving efficiency and reducing the risk of inconsistencies.
```

Comment:
I think that, as the context of this decision, we can instead insert more detailed information about why we want this feature in the first place. We idea of the _Context_ section is to make sure that any developer reading this document can understand why we the ADR was created in the first place. I don't think that only because something is _complex_ to do, then it's worth implementing a feature for it. 

Instead, we could use the following motivation as the context: 

> Currently, when a user creates a post their address is assigned the `author` role and they become the only user that can later edit or manage such post. Even though this currently works, it does not support all use cases. For example, currently we do not allow users to transfer the ownership of such post to another user. This might be particularly useful in contexts where an employee of a company might want to draft a post and later transfer the ownership of such post to their company.

Reply:
Updated to ADR